### PR TITLE
Implement a console program to calculate triangle area.

### PR DESCRIPTION
**Pull request by KaplenkoAnastasiia**

### CODE REVIEW

File: tasks/task02/task02.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+msg1 = "Enter base and height:"
+msg2 = "Enter 2 sides and angle(degrees) between them:"
+msg3 = "Goodbye!"
+
+
+def run_menu():
+    print("Welcome to the triangle area calculation tool.")
+    while True:
+        print("\n Menu:", "1. Calculate triangle area by base and height",
```
Comment (ahodkov) [inline tasks/task02/task02.py]: I think it should be in a variable

File: tasks/task02/task02.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            print(msg2)
+            calculate_area_by_angle(input())
+            continue
+        if user_string == '3':
+            print(msg3)
+            break
+        continue
+
+
+def calculate_area_by_height(userString):
```
Comment (ahodkov) [inline tasks/task02/task02.py]: `user_string`

File: tasks/task02/task02.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    values = userString.split(" ")
+    if values.__len__() != 2:
```
Comment (ahodkov) [inline tasks/task02/task02.py]: `len(values)`

File: tasks/task02/task02.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        print("incorrect values or amount of them")
+        return
+    base, height = values
```
Comment (ahodkov) [inline tasks/task02/task02.py:34]: Need positive number(or no-zero) checks

File: tasks/task02/task02.py [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    base = int(base)
+    height = int(height)
+    area = (base*height)/2
+    print(f'Area is: {area: .2f}')
+
+
+def calculate_area_by_angle(userString):
+    values = userString.split(" ")
+    if values.__len__() != 3:
```
Comment (ahodkov) [inline tasks/task02/task02.py]: same

File: tasks/task02/task02.py [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+        print('incorrect values or amount of them')
+        return
+    side1, side2, angle = values
```
Comment (ahodkov) [inline tasks/task02/task02.py:46]: same

File: tasks/task02/task02.py [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+        print("incorrect values or amount of them")
+        return
+    base, height = values
+    base = int(base)
+    height = int(height)
+    area = (base*height)/2
+    print(f'Area is: {area: .2f}')
+
+
+def calculate_area_by_angle(userString):
```
Comment (ahodkov) [inline tasks/task02/task02.py:41]: same

File: tasks/task02/task02.py [hunk 8]
```diff
@@ -0,0 +1,57 @@
+import math
+
+msg1 = "Enter base and height:"
+msg2 = "Enter 2 sides and angle(degrees) between them:"
+msg3 = "Goodbye!"
+
+
+def run_menu():
+    print("Welcome to the triangle area calculation tool.")
+    while True:
```
Comment (ahodkov) [inline tasks/task02/task02.py]: Better to use `user_input != 3`

**Review by ahodkov — CHANGES_REQUESTED** [review]

File: tasks/task02/task02.py (hunk 4 above)
Comment (KaplenkoAnastasiia) [reply tasks/task02/task02.py:34]: added

**Review by ahodkov — APPROVED** [review]

**Pull request merged**